frontend/src/components/CreateProjectDialog.tsx

```diff
@@ -25,16 +24,23 @@ function CreateProjectDialog({}:CreateProjectDialogProps) {
     const handleSubmit = async (e: React.FormEvent) => {
         e.preventDefault();
         setError(ErrorType.None);
-
-        try {
-            // await fetch POST /api/v1/project/:id/?name={projectName}&description={description};
-            closeDialog(); 
-        } catch (err) {
+        const response = await fetch(`/api/v1/project/${projectID}`, {
+            method: 'POST',
+            headers: {
```

Comment:
You need to add the Authorization header with the token that you get from a `useAuth` call. It might be easier to add the token as a prop, up to you.